src/DIRAC/Resources/Computing/InProcessComputingElement.py

```diff
@@ -89,17 +89,18 @@ def submitJob(self, executableFile, proxy=None, **kwargs):
     if not result['OK']:
       self.log.error('Fail to run InProcess', result['Message'])
     elif result['Value'][0] > 128:
+      res = S_ERROR()
       # negative exit values are returned as 256 - exit
+      res['Value'] = result['Value'][0] - 256  # yes, it's "correct"
       self.log.warn('InProcess Job Execution Failed')
-      self.log.info('Exit status:', result['Value'][0] - 256)
-      if result['Value'][0] - 256 == -2:
+      self.log.info('Exit status:', result['Value'])
+      if result['Value'] == -2:
```

Comment:
@aldbr I followed your advice but here shouldn't it be now
```suggestion
      self.log.info('Exit status:', res['Value'])
      if res['Value'] == -2:
```
?

Reply:
Yes you're right, sorry